### ISSUE TITLE
FR: OpenID Connect

### ISSUE THREAD

**Issue by blaggacao** [taigaio/taiga-back#1472]
Add Login via OpenID connect through any IdP, such as keycloac or dexidp

**Comment by blaggacao** [comment]
Please make it clear that https://github.com/taigaio/taiga-contrib-oidc-auth is a generic OICD plugin, the readme suggests fedora lock-in, which looking at the code doesn't seem to be true. Why deceive people?

**Comment by LecrisUT** [comment]
Is it compatible with Keycloak's OIDC? If so what configurations are needed there? Currently there is another plugin: https://github.com/robrotheram/taiga-contrib-openid-auth that implements OIDC but is less polished.

**Comment by lknite** [comment]
Agreed, would be nice to have some additional notes that this will work with keycloak.  Am working to set this up with keycloak now and it would be nice to know its going to work.

**Comment by lknite** [comment]
@LecrisUT were you able to get it to work with keycloak?

**Comment by lknite** [comment]
I put in a ticket on the new project site:  https://github.com/kaleidos-ventures/taiga-back/issues/66
